packages/pilot/src/components/MetricList/index.js

```diff
@@ -19,25 +19,16 @@ const MetricList = ({
     <CardContent
       className={styles.list}
     >
-      {items.map(({
-        icon,
-        title: itemTitle,
-        value,
-      }, index) => (
-        <MetricListItem
-          key={`${index}-${value}`} // eslint-disable-line react/no-array-index-key
-          icon={icon}
-          title={itemTitle}
-          value={value}
-        />
-      ))}
+      <List
```

Comment:
Estes componentes com uma prop apenas, poderiam ficar com tudo na mesma linha
```suggestion
      <List items={items} />
```